rebar.config.sample

```diff
@@ -13,12 +20,6 @@
 %% Erlang compiler options
 {erl_opts, [no_debug_info,
             {i, "myinclude"},
-            %% directories containing source files
-            {src_dirs, ["src", "src2", "src3"]},
```

Comment:
We have a test in `rebar_compile_SUITE` that uses it from `erl_opts`, `RebarConfig = [{erl_opts, [{src_dirs, ["src", "extra"]}]}],`